.github/workflows/main.yaml

```diff
@@ -1,4 +1,4 @@
-name: app
+name: app tests
```

Comment:
What means `app tests`?

Reply:
This is just a naming rectification. This workflow is intended to run tests, so the name should contain something about the fact :)